website/blog/2019-08-31-dandelion-overview.md

```diff
@@ -1,73 +1,73 @@
 
-Specifically we wanted to make sure that we met the following requirements:
+In addition to providing this core feature, we are also enabling the following features:
 
-1) Members should be able to redeem tokens for a proportional share of the organizations assets
-2) It should be possible to restrict exit actions based on whether the individual had voted yes in a pending proposal.
-3) New members should be able to propose a payment to the organization in exchange for the organizations tokens
-4) Proposals should require the proposer to lock tokens in order to ensure that the incentives of voters are aligned with their votes
-5) Users who choose to vote no, or not vote, should be able to see whether a proposal has passed before deciding to exit.
+1) Members can redeem tokens for a proportional share of the organizations assets.
+2) It is possible to restrict exit actions based on whether a member has voted yes in a pending proposal.
+3) New members are able to propose a payment to the DAO in exchange for the DAO's native tokens.
+4) Proposals require the proposer to lock tokens in order to ensure that the incentives of voters are aligned with their votes.
+5) Users who vote no, or do not vote, can see whether a proposal has passed before deciding to exit.
 
-We expected this to be fairly straightforward, but it turns out that in order to make the design work well we have had to dig into some lesser known features of aragonOS like [ACL Oracles](https://hack.aragon.org/docs/acl_IACLOracle) that to our knowledge have not been used by any other Aragon apps at this point. ACL Oracles are essentially small helper functions that plugin to Aragon's  access control list (ACL) to do more sophisticated permission evaluation. There is some information in this post about how we are using ACL Oracles, but we plan to write a more thorough and general post about how ACL Oracles can be used to make the Aragon Stack more useful. This post will focus on exploring the functionality of each app within the Dandelion org template from a high level. The apps included in the 1Hive Dandelion Org are as follows:
+We expected this to be fairly straightforward. Turns out it's not. In order to make Dandelion Orgs work we dug into some of the lesser known features of the aragonOS like [ACL Oracles](https://hack.aragon.org/docs/acl_IACLOracle). ACL Oracles are small helper functions that plug in to Aragon's  access control list (ACL) to do more sophisticated permission evaluation. ACL Oracles are a complex and powerful feature of the aragonOS. In the future we plan to write a more thorough post about how we used them and how others might do the same. This post, however, will will be a high level overview of the apps that combine to make the Dandelion Org template.
```

Comment:
` features of the aragonOS ` --> `features of aragonOS`

Reply:
changed